bin/badgekit_to_badgr/migrateBKtoBS.js

```diff
@@ -0,0 +1,144 @@
+'use strict';
+
+var sqlite3 = require('sqlite3');
+var fs = require('fs-extra');
+
+var env = (function () {
+  var Habitat = require('habitat');
+  Habitat.load('.db.env');
+  Habitat.load('default.db.env');
+  return new Habitat();
+})();
+
+// Make a Sync copy of the DB with badges to get started with
+fs.copySync('badges_only.local.sqlite3', 'local.sqlite3');
+
+// Destination DB for badge instances
+var badgesDB = new sqlite3.Database('local.sqlite3', function(err){
+  if (err) {
+    throw err;
+  }
+});
+
+// Source DB for badge instances; exported from badgekit-server
+var bkDB = new sqlite3.Database('badgekit.db', function(err){
+  if (err) {
+    throw err;
+  }
+});
+
+function lookUpBadgeData(badgesList, dataItem) {
+  var badgeData = {};
+  badgesList.forEach(function(item){
+    // A few of the slugs are different in the two databases, in some cases the difference is just an underscore, but
+    // in other cases (like the three below) the full name is different. The ones without difference are taken care of
+    // in the first if statement.
+    if (item.slug.replace(/-/g, '') === dataItem.slug.replace(/_/g, '')) {
+      badgeData = { 'image': item.image, 'id': item.id };
+    }
+
+    if (item.slug === 'funding-acquisition' && dataItem.slug === 'funding' ) {
```

Comment:
It's a bit random for an object. This can be cleanup when/if the badges data gets automated; not a priority right now, but would be good to get to it at some point instead of relying on a half manually created DB.